06week/higherOrder.js

```diff
@@ -1,31 +1,62 @@
-'use strict';
+"use strict";
 
-const assert = require('assert');
+const assert = require("assert");
 
 function forEach(arr, callback) {
   // Your code here
+  for (let count = 0; count < arr.length; count++) {
+    callback(arr[count]);
+  }
 }
 
 function map(arr, callback) {
   // Your code here
+  const newArr = [];
+  // for each item in the array, call the callback function on it
+  arr.forEach(item => {
```

Comment:
Should not be using methods. The purpose of this exercise is to teach you how to build these higher order functions.